src/Microsoft.Health.Fhir.SqlServer/Features/Watchdogs/DefragWatchdog.cs

```diff
@@ -126,7 +126,7 @@ protected override async Task ExecuteAsync()
 
         private async Task ChangeDatabaseSettingsAsync(bool isOn, CancellationToken cancellationToken)
         {
-            using var cmd = new SqlCommand("dbo.DefragChangeDatabaseSettings") { CommandType = CommandType.StoredProcedure};
+            await using var cmd = new SqlCommand("dbo.DefragChangeDatabaseSettings") { CommandType = CommandType.StoredProcedure};
```

Comment:
Why did this get added to this PR?

Reply:
True, not specifically related, though I think I looked through most of the jobs when doing this. Gotta enable those Async Disposables. Can be reverted, but seems pretty minor

Reply:
It's fine here, I just couldn't see how it was connected.